### ISSUE TITLE
Suggestion for calculating Lucky+Frenzy caution threshold on buildings/upgrades

### ISSUE THREAD

**Issue by llamsyneb** [Aktanusa/CookieMonster#30]
Right now, the Lucky+Frenzy caution threshold shows you how many more cookies you need to bank so that when you purchase a building or upgrade, you remain able to hit the maximum reward for Lucky+Frenzy.  I think this is the wrong number to be targeting.

Take a look at this screenshot: 
![screen shot 2014-10-10 at 10 44 14 am](https://cloud.githubusercontent.com/assets/1270292/4594052/197abc7e-508c-11e4-861c-e1367627a38d.png)

A maxed out Lucky+Frenzy requires 189Qa cookies in the bank for me.  I want to purchase something costing 89Qa, and the tooltip tells me I should wait until I have 265Qa more cookies (for a total of 460Qa cookies).  If I do wait until I have 460Qa cookies, then when I purchase Kitten Overseers for 89Qa, I'll have 371Qa cookies in the bank, which will be the new minimum bank to continue maxing out Lucky+Frenzy.

However, I believe the real target, instead of being 460Qa, should be 278Qa (the cost of the upgrade + my _current_ required Lucky+Frenzy cookie bank).  If I buy Kitten Overseers when I have 278Qa, then my bank will be at my _current_ Lucky+Frenzy minimum (189Qa), and my _new_ minimum will be considerably higher because of the upgrade's CPS boost (371Qa).  So, when I get Lucky+Frenzy now, I will clearly not be getting the maximum reward possible.  However, my reward will be greater than if I had waited to buy the upgrade, because my bank will be somewhere between 189Qa and 371Qa (the new required minimum).

In short, waiting until you have enough in the bank to _stay_ at the Lucky+Frenzy bank minimum is leaving cookies on the table (in the jar?).  The Lucky+Frenzy caution should direct you to wait till you have your pre-upgrade required bank + the cost of the upgrade, not the post-upgrade required bank + the cost of the upgrade.
